test/run/test.run.js

```diff
@@ -115,3 +116,23 @@ describe("fx-runner start", function () {
     });
   });
 });
+
+describe("concat binary arguments", function () {
+  it("binary arguments as string", function (done) {
+    var arr = concatBinArgs([], "-a b -c \"d e\"");
+    expect(arr[0]).to.be.equal("-a");
+    expect(arr[1]).to.be.equal("b");
+    expect(arr[2]).to.be.equal("-c");
+    expect(arr[3]).to.be.equal("\"d e\"");
```

Comment:
Shouldn't this equal a string of  `d e` without escaped quotes? I'm looking at the [args](https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options) array passed to `spawn()` and I'm pretty sure that we don't want to feed it quotation marks. Did you test this against a real binary?